apis/kubedb/v1alpha2/zookeeper_helpers.go

```diff
@@ -235,6 +262,66 @@ func (z *ZooKeeper) assignDefaultContainerSecurityContext(zkVersion *catalog.Zoo
 	}
 }
 
+func (z *ZooKeeper) assignDefaultInitContainerSecurityContext(rmVersion *catalog.ZooKeeperVersion, sc *core.SecurityContext) {
```

Comment:
Is there any reason to handle the initContainer securityContext seperately ?

Reply:
Updated